src/openzaak/components/documenten/admin.py

```diff
@@ -37,10 +38,34 @@ class GebruiksrechtenAdmin(AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmi
     viewset = viewsets.GebruiksrechtenViewSet
 
 
+class ObjectInformatieObjectForm(forms.ModelForm):
+    class Meta:
+        model = ObjectInformatieObject
+        fields = "__all__"
+
+    def clean(self):
+        cleaned_data = super().clean()
+
+        if not cleaned_data.get("_zaak") and not cleaned_data.get("_zaak_url"):
```

Comment:
In `ObjectInformatieObject` the required field is determined based on `object_type` field, i.e. `zaak` field is required only if `object_type = 'zaak'`. The same situation is with `besluit` field. You can see the `CheckConstraint` in the `ObjectInformatieObject` model as an example. The check in the admin should be consistent with this logic.

Reply:
Ya, I didn't realize some of the `FkOrUrlFields` are not required.  I think the first few I saw were required so I mistakenly assumed they all were.  I'll double check all of them and only add checks for the ones that are definitely required. :)